src/pages/messages/components/TicketListItem.tsx

```diff
@@ -106,7 +104,7 @@ export const TicketListItem = ({ ticket }: TicketListItemProps) => {
             gap: '0 1rem',
             gridTemplateColumns: { xs: '1fr', sm: '10fr 4fr 2fr 2fr 1fr' },
           }}>
-          <RegistrationListItemContent registration={registrationSearchItem} ticketView />
+          <RegistrationListItemContent registration={registrationSearchItem} ticketView ticketType={ticket.type} />
```

Comment:
Added ANOTHER parameter to this super bloated component. It really really needs to be divided into two. (But it was necessary to get right background on ticket type icon)